src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/xml/Elements.xsd

```diff
@@ -23,7 +23,7 @@
                   <!-- The namespace in which the element appears (e.g., "System.Windows") -->
                   <xs:attribute name="Namespace" type="xs:string" />
                   <!-- The path where the generator should output the code
-                       relative to %sdxroot%\wpf (e.g., "src\Core\CSharp\System\Windows\Generated")
```

Comment:
We should replace references of `%sdxroot%\wpf` as well.

Reply:
Yes, SdxRoot is only found in comments but it doesn't hurt to fix them as well.